index.js

```diff
@@ -112,7 +119,7 @@ function _init() {
   }
 
   eyes.defaultWindowSize = null;
-  eyes.setBatch(appName, getBatchUUID());
```

Comment:
Notice that when running locally it will always run the older `setBatch` configuration (unless whoever uses it adds the `APPLITOOLS_BATCH_ID` env variable).
Does this matter for us?
Should we add this to the README?

Reply:
you can add a small README not to add the `APPLITOOLS_BATCH_ID` before running the steps
something like

```json
{
  "scripts": {
    "test": "APPLITOOLS_BATCH_ID=something yoshi test"
  }
}
```